### PR TITLE
Update Libraries, Add /healthz endpoint

### PR DESCRIPTION
**Pull request by cpressland**
Update all Libraries to latest versions, add a /healthz endpoint. Replaced logging library with zerolog as `github.com/prometheus/common/log` has since been removed.

### CODE REVIEW

**Comment by xbglowx** [comment]
Sorry, just noticed this. I will take a look.

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.0.5
+0.0.6
```
Comment (xbglowx) [inline VERSION:1]: Can you please leave this out. I will update once I am ready to cut a new release.

**Review by xbglowx — CHANGES_REQUESTED** [review]
I believe you will need call Msg or Msgf in order for fatal to work properly and exit. For example, as is, if you don't specify environment variable `MG_API_KEY`, it will not exit. 

The screenshot was taken from [here](https://pkg.go.dev/github.com/rs/zerolog/log@v1.23.0#Fatal)

Screen Shot 2021-09-29 at 9 46 54 AM

Replacing `log.Fatal().Err(err)` with `log.Fatal().Err(err).Msgf("%v", err)` should work.

**Comment by cpressland** [comment]
Requested changes made, let me know if you want anything else.

**Comment by Muhammadmohid** [comment]
can you please give me a guide how to add my domain and do i have to add my api key? and how to add those kindly change the main guide

**Pull request merged**